### PR TITLE
Fix according to breaking changes in `ark-ec`

### PR DESCRIPTION
**Pull request by autquis**
- Fixes due to arkworks-rs/algebra#746 
- Removing the extra loop mentioned in #139 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (master)
- [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [ ] Wrote unit tests
- [ ] Updated relevant documentation in the code
- [ ] Added a relevant changelog entry to the `Pending` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

**Review by mmagician — APPROVED** [review]

File: poly-commit/src/marlin/marlin_pst13_pc/mod.rs [hunk 1]
```diff
@@ -210,47 +210,31 @@ where
...
-            FixedBase::msm::<E::G1>(scalar_bits, window_size, &g_table, &powers_of_beta);
-        powers_of_g.push(g);
+        let mut powers_of_g = g.batch_mul(&powers_of_beta);
+        powers_of_g.push(g.into_affine());
         powers_of_beta_terms.push(P::Term::new(vec![]));
         end_timer!(g_time);
 
         let gamma_g_time = start_timer!(|| "Generating powers of gamma * G");
-        let window_size = FixedBase::get_mul_window_size(max_degree + 2);
-        let gamma_g_table = FixedBase::get_window_table(scalar_bits, window_size, gamma_g);
```
Comment (Pratyush) [inline poly-commit/src/marlin/marlin_pst13_pc/mod.rs:225]: We should keep the table, because it's used in every loop below.

**Review by Pratyush — APPROVED** [review]
Looks good modulo two questions!

File: poly-commit/src/marlin/marlin_pst13_pc/mod.rs (hunk 1 above)
Comment (mmagician) [reply poly-commit/src/marlin/marlin_pst13_pc/mod.rs:225]: Good catch, fixed in a270950cb5e7ffa334955a2799f3b0094796b8a2

**Review by mmagician — COMMENTED** [review]
There was another place where preprocessing table was not used (even before this PR). fixed in f23de5d9f84fc232e5bece831a085adb2471ec9a

**Comment by Pratyush** [comment]
Lovely, thanks!

**Review by Pratyush — APPROVED** [review]

**Pull request merged**